CombatLogger/src/Lambo/CombatLogger/Main.php

```diff
@@ -48,6 +49,21 @@ public function EntityDamageEvent(EntityDamageEvent $event){
         }
     }
 
+   public function onCommandExecute(PlayerCommandPreprocessEvent $event) {
+         $command = $event->getMessage();
+         $commandarray = explode(' ',trim($command));
+         $message = $commandarray[0];
+    if(isset($this->players[$event->getPlayer()->getName()])){
+      if (in_array($message, $this->getConfig()->get("Blocked-Commands"))) {
```

Comment:
Huh? They can use the config to set the commands.
On 5 Jun 2015 16:26, "PEMapModder" <notifications@github.com> wrote:

> In CombatLogger/src/Lambo/CombatLogger/Main.php
> <https://github.com/Lambo16/CombatLogger/pull/1#discussion_r31822250>:
>
> > @@ -48,6 +49,21 @@ public function EntityDamageEvent(EntityDamageEvent $event){
> >          }
> >      }
> >
> > +   public function onCommandExecute(PlayerCommandPreprocessEvent $event) {
> > +         $command = $event->getMessage();
> > +         $commandarray = explode(' ',trim($command));
> > +         $message = $commandarray[0];
> > +    if(isset($this->players[$event->getPlayer()->getName()])){
> > +      if (in_array($message, $this->getConfig()->get("Blocked-Commands"))) {
>
> yes it works, but not very customizable.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Lambo16/CombatLogger/pull/1/files#r31822250>.
>